### DISCUSSION TITLE
binlog的过滤是在数据源端还是在本地过滤

### DISCUSSION THREAD

**Discussion by BobbySun** [apache/flink-cdc#582] [General] [answered]
mysql CDC 指定表同步，binlog的过滤是将库的所有binlog同步到本地后再过滤吗？

**Comment by wuchong** [comment, marked as answer]
在本地过滤。主要是 mysql 不支持在 server 端过滤。

**Reply by lykpass** [reply]
如果我的账号权限是到表级别的，这个实例的其它表的日志还会到本地吗

**Reply by wuchong** [reply]
读 binlog 权限 `REPLICATION SLAVE` 是一个 Administrative privileges，是全局的，不针对具体数据库的。

> Administrative privileges enable users to manage operation of the MySQL server. These privileges are global because they are not specific to a particular database.

https://dev.mysql.com/doc/refman/8.0/en/privileges-provided.html
